inmap/cmd/config.go

```diff
@@ -119,14 +121,24 @@ func ReadConfigFile(filename string) (config *ConfigData, err error) {
 	}
 
 	for k, v := range config.OutputVariables {
+		v = strings.Replace(v, "\r\n", " ", -1)
```

Comment:
Yeah I guess I was avoiding creating two spaces where there should only be one, but I don't think that actually matters.